src/changeDataStructure.js

```diff
@@ -0,0 +1,24 @@
+import {OrderedMap, Map} from 'immutable';
+
+export function arrToMap(arr, DataRecord = Map) {
```

Comment:
What is the benefit of this normalization of data, and why this function re-used from here ? https://github.com/bars/react-learnjs/blob/master/js_ru_08_06_17-master/src/helpers.js

Reply:
i studied this course and used it because a record is more convenient that list

Reply:
Thanks for your explanation